examples/semiconductor/diode_BAS16_output.json

```diff
@@ -0,0 +1,272 @@
+{
+    "componentID": {
+        "partType": "diode",
+        "manufacturer": "Nexperia",
+        "componentName": "BAS16",
+        "orderableMPN": [
+            "BAS16"
```

Comment:
Since we have Nexperia's feedback on BAS16, we can add before releasing 1.0

Reply:
done